pkg/controller/controller_binding.go

```diff
@@ -221,6 +221,30 @@ func (c *controller) reconcileBinding(binding *v1alpha1.Binding) error {
 
 		response, err := brokerClient.Bind(request)
 		if err != nil {
+			if osb.IsHTTPError(err) {
+				httpErr := err.(osb.HTTPStatusCodeError)
+				s := fmt.Sprintf("Error creating Binding \"%s/%s\" for Instance \"%s/%s\" of ServiceClass %q at Broker %q with status code %d: ErrorMessage: %v, Description: %v",
+					binding.Name,
+					binding.Namespace,
+					instance.Namespace,
+					instance.Name,
+					serviceClass.Name,
+					brokerName,
+					httpErr.StatusCode,
+					*httpErr.ErrorMessage,
```

Comment:
That's true, but seeing a pointer address isn't all that helpful. See if you like https://github.com/pmorie/go-open-service-broker-client/pull/44 and I can adjust to use the error method instead.